README.md

```diff
@@ -1614,9 +1614,9 @@ See also [Are we game yet?](https://arewegameyet.rs)
 * [tokio-rs/tracing](https://github.com/tokio-rs/tracing) - An application level tracing framework for async-aware structured logging, error handling, metrics, and more [![Build Status](https://github.com/tokio-rs/tracing/workflows/CI/badge.svg?branch=master)](https://github.com/tokio-rs/tracing/actions?query=workflow%3ACI)
 
 ### Macro
-
 * cute
```

Comment:
Btw, I also noticed, that the crate `cute` hasn't been updated for 7 years. Should it be removed?

Reply:
If it doesn't work anymore, yes, but if it's still working it's all fine.

Reply:
It works, so I won't remove it.